GIT_CONVENTIONS.md

```diff
@@ -0,0 +1,87 @@
+# Git Conventions for the ORAS Community
+
+This guide is a specification for adding human and machine readable meaning to Pull Request and git commit messages in the ORAS community. It provides an easy set of rules for creating an explicit commit history; which makes it easier to explore a more structured commit history and automatically generating CHANGELOGs for each new release. 
+
+When contributing to the ORAS projects, please first discuss the change you wish to make via a GitHub issue, a GitHub discussion, or any other methods with the ORAS community maintainers before making a change.
```

Comment:
This line sounds like it's part of a CONTRIBUTING.md. Akin to how there is friendly reminder about the Code of Conduct below, should we link to that here and put the two references in the same indented section? E.g.,

> As a reminder, please follow the [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) in all your interactions with the ORAS project, and follow the [reviewers guide](https://github.com/oras-project/community/blob/main/REVIEWING.md) when contributing to an ORAS project.

Reply:
Nice catch! I've already added a link to `CODE_OF_CONDUCT.md` in line 7.
`> Note: Please follow the [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) in all your interactions with the ORAS project.`

and yes, I think we could also have a reference to reviewers guide here. Thanks for pointing this out!